README.md

```diff
@@ -1338,9 +1338,9 @@ expect(Paths.get("/usr/bin/noprogram")).exists()
 ```
 ↑ <sub>[Example](https://github.com/robstoll/atrium/tree/master/samples/readme-examples/src/main/kotlin/readme/examples/ReadmeSpec.kt#L326)</sub> ↓ <sub>Output</sub>
 ```text
-expect: /usr/bin/noprogram        (sun.nio.fs.UnixPath <1234789>)
+expect: \usr\bin\noprogram        (sun.nio.fs.WindowsPath <1234789>)
```

Comment:
please revert the changes in this file

Reply:
Okay I reverted that, will be up in the next push